### PR TITLE
Add R package rlist

### PR DESCRIPTION
**Pull request by inodb**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by inodb** [comment]
@bioconda/core I added this package because I've seen it used in some bioinf scripts. Not sure if this is the best place to add though, maybe it should be in the anaconda r channel since it's not bioinf specific? Thanks

**Comment by bgruening** [comment]
You could put it into the conda-forge channel: https://github.com/conda-forge/
But I think it is also fine to put it here for the time being.

**Comment by inodb** [comment]
@bgruening Great, thanks! Can I merge this then? Not sure if this qualifies as a review

File: recipes/r-rlist/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "0.4.6.1"
+
+source:
+  fn: rlist_0.4.6.1.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/rlist_0.4.6.1.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/rlist/rlist_0.4.6.1.tar.gz
+
```
Comment (bgruening) [inline recipes/r-rlist/meta.yaml]: Can we add a checksum here?

File: recipes/r-rlist/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  license: MIT + file LICENSE
+  summary: Provides a set of functions for data manipulation with list objects, including mapping,
+    filtering, grouping, sorting, updating, searching, and other useful functions. Most
+    functions are designed to be pipeline friendly so that data processing with lists
+    can be chained.
+  license_family: MIT
+
+# The original CRAN metadata for this package was:
+
+# Package: rlist
```
Comment (bgruening) [inline recipes/r-rlist/meta.yaml]: We should fix the skeleton generator, but if you feel like it, please clean up the comments - not strictly needed.

**Comment by bgruening** [comment]
Looks good to me, please add a checksum and feel free to merge :)

File: recipes/r-rlist/meta.yaml (hunk 2 above)
Comment (inodb) [inline recipes/r-rlist/meta.yaml]: As in, remove it?

Comment (bgruening) [inline recipes/r-rlist/meta.yaml]: yes :)

**Pull request merged**